nrrd/tests/test_writing.py

```diff
@@ -228,7 +230,7 @@ def test_write_detached_ascii(self):
         data, header = nrrd.read(output_filename)
```

Comment:
Sorry, one small thing. Since you directly placed the string to be asserted, the `output_data_filename` variable is not used. Remove that please

Same for above.

Reply:
Will review your other PRs, but may take some time.